src/DataProvider.js

```diff
@@ -136,19 +161,45 @@ export default {
     }
     url += this.authQueryString();
 
-    return this.post(url, params, { timeout: 0});
+    return this.post(url, params, { timeout: 0 });
+  },
+
+  /**
+   * Gets data source service
+   * @param {number|null} dataSourceId
+   * @param {object} params
+   * @returns {object}
+   */
+  getDataSource(dataSourceId, params) {
+    let url = `/requests/data_sources/${dataSourceId}/resources/${params.config.endpoint}/data`;
+    url += this.authQueryString();
+    return this.get(url, {
+      useCache: window.ProcessMaker.screen.cacheEnabled,
+      params: params.config.outboundConfig
```

Comment:
Missing `params.data`.
The endpoint `/api/1.0/requests/data_sources/{data_source_id}` could receive the following parameters:

In GET: `?queryString=some_string`
In POST: ` config: { object } `
In POST: ` data: { object } `

Since the `config` and `data` could be complex objects, consider to encode with JSON the parameters `config` and `data`. Something like:

`?config={"params":{"foo":"bar"}}&data={"foo":"bar"}`